solutions/standard/main.tf

```diff
@@ -68,9 +68,16 @@ module "event_notifications" {
   service_endpoints        = var.service_endpoints
   service_credential_names = var.service_credential_names
   # KMS Related
-  kms_encryption_enabled        = true
-  kms_endpoint_url              = var.kms_endpoint_url
-  existing_kms_instance_crn     = local.existing_kms_instance_crn
-  root_key_id                   = local.en_kms_key_id
-  skip_iam_authorization_policy = var.skip_en_kms_auth_policy
+  kms_encryption_enabled    = true
+  kms_endpoint_url          = var.kms_endpoint_url
+  existing_kms_instance_crn = local.existing_kms_instance_crn
+  root_key_id               = local.en_kms_key_id
+  skip_en_kms_auth_policy   = var.skip_en_kms_auth_policy
+  # COS Related
```

Comment:
The DA should support created a new COS instance and/or bucket, as well as optionally passing existing COS. Please take same approach as https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/blob/main/solutions/instances

Reply:
Also please update the `solutions/standard/README.md` with the details of the new feature